src/PeopleSDKAdapter.test.js

```diff
@@ -119,9 +121,44 @@ describe('People SDK Adapter', () => {
     });
   });
 
+  describe('getPersonList', () => {
+    test('returns an observable', () => {
+      expect(isObservable(peopleSDKAdapter.getPeopleList(query))).toBeTruthy();
+    });
+
+    test('emits persons list on subscription', (done) => {
+      peopleSDKAdapter.getPeopleList(query).subscribe((peopleList) => {
+        expect(peopleList).toEqual([{
+          id: 'personIDCurrentUser',
+          emails: ['email@cisco.com'],
+          displayName: 'sipUri',
+          firstName: 'Webex',
+          lastName: 'Components',
+          avatar: 'avatar',
+          orgId: 'orgID',
+        }]);
+      });
+      done();
+    });
+
+    test('throw error on fetching person list', (done) => {
```

Comment:
getPeopleList does not throw an error, it returns an observable that emits either an array (people list) or an error.

'emits an error if the SDK fails to fetch the person list'